api/proto/policy.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package protocols;
+
+option go_package = "github.com/oscal-compass/compliance-to-policy-go/v2/api/proto/";
+
+import "api/proto/messages.proto";
+
+// get policy results from PVP
+service PolicyEngine {
+  rpc GetResults(Empty) returns (ResultsResponse);
```

Comment:
I think we might need to update this to include `repeated Rules` as an input as well if we considered Generation and Policy to be two completely separate operation. What do you think?